### ISSUE TITLE
App partial use the safe area of my Pixel 5

### ISSUE THREAD

**Issue by xelayz** [organicmaps/organicmaps#2515]
Hello,

I use OrganicMaps on my Pixel 5 with Android 12 and the app partial use the safe area. This looks not good and the app should respect the space containing my notch. On my Mi 8 with LineageOS Android 10 it is working fine.

Pixel 5: https://pic.xelayz.com/images/bda479bea23857c80420f0c039fa0e02.jpg
Mi 8: https://pic.xelayz.com/images/89756fcb535a58791a055c91d9d3c718.jpg

**Comment by biodranik** [comment]
Is it about a camera and layers button, or about something else?

**Comment by xelayz** [comment]
Yes. The line between the dark area and the map should be under the camera. I mean this line: https://pic.xelayz.com/images/2c2958e2f0cd605ab40bf561e37fed15.png On other phones OrganicMaps do not use the safe area. And on my Pixel 5 the camera is for sure a part of the safe area. But a part of the normal map, without the dark overlay, is behind the camera. Also the layers button.

**Comment by biodranik** [comment]
Can you please show how other apps (maps?) are properly displayed on your screen?

**Comment by xelayz** [comment]
I added a screenshot of OSMAnd. I think this is the best to compare with. But there are some other apps they have problems with the safe area of my Pixel 5. Like the spotify search. Checkout the screenshot. But also a lot apps do not have problems and everything is great, like OsmAnd.

![Screenshot_20220506-035107.png](https://user-images.githubusercontent.com/59479772/167053887-6f2c9a5a-4f28-49e4-b5f5-0c7f5b1a09cd.png)

![Screenshot_20220506-035343.png](https://user-images.githubusercontent.com/59479772/167054173-506a6a86-2670-4414-9d51-22812de37711.png)

**Comment by strump** [comment]
@xelayz links to images are not working anymore :(
* https://pic.xelayz.com/images/bda479bea23857c80420f0c039fa0e02.jpg
* https://pic.xelayz.com/images/89756fcb535a58791a055c91d9d3c718.jpg
* https://pic.xelayz.com/images/2c2958e2f0cd605ab40bf561e37fed15.png 

Could you please upload those images directly to Github?

**Comment by biodranik** [comment]
Probably @arnaudvergnet 's refactoring fixed it.

**Comment by arnaudvergnet** [comment]
> Probably @arnaudvergnet 's refactoring fixed it.

I don't remember changing anything related to this, maybe this was fixed by using a more recent version of the Android SDK as target?
